src/main/java/eu/dissco/orchestration/backend/controller/SourceSystemController.java

```diff
@@ -25,45 +31,53 @@
 
 @Slf4j
 @RestController
-@RequestMapping("/source-system")
+@RequestMapping("orchestrator/source-system")
```

Comment:
Why ochestrator? I don't think this is necessary as we can put that in the route path and let the application work from there.
The path will then be `/api/v1/orchestration/source-system` (for example) and the application only receives `/source-system`